docs/docs/accessibility.md

```diff
@@ -0,0 +1,320 @@
+---
+id: accessibility
+title: Accessibility
+permalink: docs/accessibility.html
+redirect_from:
+  - "docs/index.html"
+---
+
+## Why accessibility?
+
+Web accessibility (also referred to as [**a11y**](https://en.wiktionary.org/wiki/a11y)) is the design and creation of websites that can be used by everyone. People who experience disabilities only fully benefit from websites when 
```

Comment:
"People who experience disabilities only fully benefit from websites when they include accessibility features".

This phrase feels a bit contentious. I can't come up with a phrasing that doesn't have "othering" connotations or undertones of condescension. Let's just stick with a factful sentence about the technology as written above.